src/app.nit

```diff
@@ -15,6 +15,13 @@
 import api
 
 var opts = new AppOptions.from_args(args)
+
+if opts.opt_help.value == true then
+    print("Usage: app [Options]\nOptions:")
+    opts.usage
+    exit(0)
```

Comment:
From what I remember, @privat advises against `exit(0)` in favor of a `return 0`